client/my-sites/plan-features/style.scss

```diff
@@ -1,3 +1,33 @@
+$header-banner-height: 20px;
```

Comment:
> lets move this under the .plan-features scope so the sass variable name is not global

I'm just trying to make it work but when a Sass var is defined in a scope, it can't be used in another scope. In Calypso, we don't write CSS selectors like `.plan-features .header-banner` but like `.plan-features__header-banner`, therefore putting `$header-banner-height` under `.plan-features` scope won't work (unless I'm missing some Calypso magic :sparkles: ).

![selection_019](https://cloud.githubusercontent.com/assets/4988512/16147877/4804ec10-3486-11e6-86e4-481258999dc4.jpg)